### DISCUSSION TITLE
⏳ Add alternate template on carousel loop or swiping options to grid loop

### DISCUSSION THREAD

**Discussion by tomgacz** [elementor/elementor#24580] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I can’t add static carousel item for carousel loop

### Describe the solution you'd like.

Add alternate template function like on grid loop

### Describe alternatives you've considered.

Make grip loop swiping like on carousel loop

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by AladdinShazly** [comment]
hope this comes it's so important

**Reply by tomgacz** [reply]
I have given up hope that this will ever be implemented :/

**Comment by JoshPNG** [comment]
bump

**Comment by newuser0111** [comment]
This is a must

**Comment by moonpoolco** [comment]
Yes please

**Comment by JoshPNG** [comment]
Bump

**Comment by wearepleh** [comment]
With this feature, it would be really easy to create a coverflow effect slider if you can fit the alternate template in a fixed position of the slider, preferably the one in the center. Hope we can see it soon!

**Comment by djaysan** [comment]
Its Ridiculous that they didn't include that.
I'm updating a site using ele custom skin - it had this feature & elementor team made it obsolete leaving the author no choice to abandon this plugin. 
Now that it's a core function of elementor, it's even missing a basic function like this.
Here is what I had to do to overcome this: 
create a nested carousel with say 4-6 slides, and copy pasting my loop grid set to 1 post 1 column inside and alternating between template 1 and 2... then set the loop grid > query >  exclude: avoid duplicate. what a huge detour but at least it does the job...

**Comment by keepsmilyn** [comment]
Yes! This would be amazing. Its great that you can do it on Loop Grids, but Loop Carousels with alternate templates would be even better.

**Comment by MarcCeleiro** [comment]
We’ve built a small plugin at our studio that enables alternate templates in the Loop Carousel widget.

You’ll find the code in this gist: https://gist.github.com/MarcCeleiro/03dcbbfe5b3455bb18e0b2cb2d1b63e0

We’ve tested it in a couple of our own projects, and it’s been working reliably so far. We’ll keep refining it, but the core behavior and workflow are already in a good place. 🙂

(Ping @nicholaszein)

**Reply by wyatt98-byte** [reply]
You are an absolute life saver! @MarcCeleiro

**Comment by nicholaszein** [comment]
Keep voting guys. 😉

**Comment by jonboxkc** [comment]
This would be a great feature to add!
